src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Node Configuration
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Subcommands](#subcommands)
+- [Configuration File Options](#configuration-file-options)
+  - [Section: node](#section-node)
+  - [Section: events_observer (optional)](#section-events_observer-optional)
+  - [Section: connection_options (optional)](#section-connection_options-optional)
+  - [Section: burnchain](#section-burnchain)
+  - [Section: mstx_balance](#section-mstx_balance)
+
+## Installation
```

Comment:
Now that this page is a reference, we can probably remove this installation piece. For best results, it would likely be nice to update the tutorial referenced here though. Something at the end of the tutorial that would say "for a full reference of Stacks Node commands and config options, check out ..."